[2022-01-04 23:38:24,770] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: example_bash_operator.runme_2 manual__2022-01-05T02:37:22.196874+00:00 [queued]>
[2022-01-04 23:38:24,780] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: example_bash_operator.runme_2 manual__2022-01-05T02:37:22.196874+00:00 [queued]>
[2022-01-04 23:38:24,780] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-01-04 23:38:24,780] {taskinstance.py:1239} INFO - Starting attempt 1 of 1
[2022-01-04 23:38:24,780] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-01-04 23:38:24,889] {taskinstance.py:1259} INFO - Executing <Task(BashOperator): runme_2> on 2022-01-05 02:37:22.196874+00:00
[2022-01-04 23:38:24,892] {standard_task_runner.py:52} INFO - Started process 11504 to run task
[2022-01-04 23:38:24,898] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'example_bash_operator', 'runme_2', 'manual__2022-01-05T02:37:22.196874+00:00', '--job-id', '10', '--raw', '--subdir', '/home/rhavel/Documentos/DEPLOY-PROJECTS/first-airflow/venv/lib/python3.9/site-packages/airflow/example_dags/example_bash_operator.py', '--cfg-path', '/tmp/tmpfsd4nkk8', '--error-file', '/tmp/tmpzo1bywgl']
[2022-01-04 23:38:24,899] {standard_task_runner.py:77} INFO - Job 10: Subtask runme_2
[2022-01-04 23:38:25,056] {logging_mixin.py:109} INFO - Running <TaskInstance: example_bash_operator.runme_2 manual__2022-01-05T02:37:22.196874+00:00 [running]> on host rhavel-VPCEG13EB
[2022-01-04 23:38:25,271] {taskinstance.py:1424} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=example_bash_operator
AIRFLOW_CTX_TASK_ID=runme_2
AIRFLOW_CTX_EXECUTION_DATE=2022-01-05T02:37:22.196874+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-01-05T02:37:22.196874+00:00
[2022-01-04 23:38:25,272] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-01-04 23:38:25,273] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo "example_bash_operator__runme_2__20220105" && sleep 1']
[2022-01-04 23:38:25,282] {subprocess.py:85} INFO - Output:
[2022-01-04 23:38:25,285] {subprocess.py:89} INFO - example_bash_operator__runme_2__20220105
[2022-01-04 23:38:26,287] {subprocess.py:93} INFO - Command exited with return code 0
[2022-01-04 23:38:26,396] {taskinstance.py:1267} INFO - Marking task as SUCCESS. dag_id=example_bash_operator, task_id=runme_2, execution_date=20220105T023722, start_date=20220105T023824, end_date=20220105T023826
[2022-01-04 23:38:26,558] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-01-04 23:38:26,979] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
